common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1559,6 +1569,19 @@ def can_toggle_course_pacing(self):
         if not self.start:
             return False
         return datetime.now(utc) <= self.start
+    
+    @property
+    def enable_discussion(self):
+        return self.enable_discussion_flag
+    
+    @enable_discussion.setter
+    def enable_discussion(self, value):
+        # If we enable discussion at course level, it wouldn't cascade. But, if we disable it, it would.
+        if value is False:
+            for child in self.get_children():
+                child.enable_discussion = value            
```

Comment:
There is no need for this. This doesn't need a getter or setter. It is a simple boolean value that can set like any other field.